src/riak_kv_crdt.erl

```diff
@@ -546,6 +580,22 @@ get_context(Type, Value) ->
         false -> <<>>
     end.
 
+%% @doc Helper fun to create crdt operation.
```

Comment:
Pretty sure this is from the bigset branch, but I'm OK with it being here. Just curious how it got here :D